index.html

```diff
@@ -26,24 +26,181 @@
 
   <section class="container">
        
-    <div class=" box name"><h3>Samuel Ackon</h3></div>
-    <div class=" box intro"><h4>Hello! I am a software developer! 
+    <div class=" box name"><h1>Samuel Ackon</h1></div>
+    <div class=" box intro"><p>Hello! I am a software developer! 
         I can help you build a product, feature or website. 
         Taka a look of my works. 
         If you like what you see and have a 
         project you need coded,
-         don’t hesitate and contact me</h4></div>
+         don’t hesitate and contact me</p></div>
        <div class="rectangle">
           <div>Start collaboration</div></div>
 
        <div class="designs">
           <div class="image3"><img src="assets/image_geometry_2.png" alt="#" class="shape002"></div>
```

Comment:
- [ ] Please, consider that the section has the proper background image (decorations are always backgrounded images, not `img` tags). So you can do it in `.css` file using `background-image: url("your img path");`

Reply:
This issue has not been resolved.